garfield/lookup/views.py

```diff
@@ -0,0 +1,94 @@
+#from celery import chain
+#from celery import shared_task
+
+#from django.conf import settings
+#from django.db.models.signals import post_save
+#from django.dispatch import receiver
+#from django.forms.models import model_to_dict
+#from django.template.loader import render_to_string
+
+import requests
+#import sys
+
+#from django.shortcuts import render
+from django.http import HttpResponse
+
+from twilio.rest import Client
+from twilio.twiml.messaging_response import MessagingResponse
+
+from contacts.models import Contact
+from .decorators import twilio_view
+from garfield import local as local
+import phonenumbers
+# Create your views here.
+@twilio_view
+def index(request):
+    """
+        Endpoint to lookup contact information via Twilio
+    """
+    response = MessagingResponse()
+    message = ""
+    try:
+      parsed_data = lookup_contact(request)
+    except InputError as e:
+      message = str(e.message)
+      response.message(message)
+      return response
+    except Contact.DoesNotExist as contacexp:
+        response.message("Contact was not found")
+        return response
+    message += "Number of Texts: %d\n" % (parsed_data['num_texts'])
+
+    message += "Number of Calls: %d\n"% (parsed_data['num_calls'])
+         
+    message += "Number of Contacts Corresponded With: %d\n"% (parsed_data['suspect_contact_count'])
+    if parsed_data['suspect_carrier'] != None:
+      message += "Carrier:  " + parsed_data['suspect_carrier']
+    response.message(message)
+    return response
+
+def lookup_contact(request):
```

Comment:
We do have methods that do this for Whitepages Pro and NextCaller in `contacts/tasks.py`. Suspect we'll want to keep this DRY and use the existing ones.

Reply:
will use this method

Reply:
just bookkeeping here - we agreed on the call to keep the original implementation